numbat/src/bytecode_interpreter.rs

```diff
@@ -185,13 +193,29 @@ impl BytecodeInterpreter {
                 self.compile_expression_with_simplify(expr)?;
                 self.vm.add_op(Op::Return);
             }
-            Statement::DefineVariable(identifier, expr, _type_annotation, _type) => {
-                self.compile_expression_with_simplify(expr)?;
+            Statement::DefineVariable(identifier, decorators, expr, _type_annotation, _type) => {
                 let current_depth = self.current_depth();
-                self.locals[current_depth].push(Local {
-                    identifier: identifier.clone(),
-                    depth: 0,
-                });
+
+                // For variables, we ignore the prefix info and only use the names
```

Comment:
Should we allow disallow adding prefix information in the parser?

Reply:
Agreed.  I just pushed a commit that disallows this in the parser